docker-compose.katana.yaml

```diff
@@ -2,8 +2,8 @@ version: "3.2"
 
 services:
   starknet:
-    image: ghcr.io/dojoengine/dojo:v0.1.0
-    command: 
+    image: ghcr.io/dojoengine/dojo:latest
```

Comment:
This [pr](https://github.com/dojoengine/dojo/pull/685) got merged before the [latest nightly](https://github.com/dojoengine/dojo/pkgs/container/dojo) was released so it won't have the `curl` changes. For now I'll put latest here so when the new image is released, the `healthcheck` in Katana will work